packages/grid/_modules_/grid/components/column-headers.tsx

```diff
@@ -62,17 +82,27 @@ export const ColumnsHeader = React.memo(
         }
       }, [renderCtx, columns]);
 
+      const onDragOver =
```

Comment:
How we usually prefix the handler

```suggestion
      const handleDragOver =
```

Reply:
Done. It was my bad.